src/hw1/ParallelExternalLongSorter.java

```diff
@@ -307,7 +324,12 @@ public Void call() throws Exception {
             scratch.mark();
             long[] tmp = new long[input.remaining()];
             input.get(tmp);
-            Arrays.sort(tmp);
+            if (sortingMethod == "parallel" ) {
```

Comment:
I think passing around "mode"/"type" strings is a code smell; it would make more sense to extend ChunkSorter... 

It's also not clear what the utility of this class is since we're not allowed to use parallelSort when our solution is being evaluated by the professor- unless the purpose is for establishing a baseline benchmark to compare our performance against. In that case, it makes even more sense to divorce this behavior from our implementation by extending ChunkSorter

Reply:
it's just for a baseline benchmark